indexing-service/src/main/java/org/apache/druid/indexing/common/actions/RetrieveUsedSegmentsAction.java

```diff
@@ -73,6 +73,9 @@ public class RetrieveUsedSegmentsAction implements TaskAction<Collection<DataSeg
   @JsonIgnore
   private final List<Interval> intervals;
 
+  @JsonIgnore
+  private final List<String> versions;
```

Comment:
This action does not need a list of versions. Please see the comment on `KillUnusedSegmentsTask`.